ncm-modprobe/src/main/pan/components/modprobe/schema.pan

```diff
@@ -11,7 +11,8 @@ type module_type = {
     "alias"     ? string # alias for the loadable module"
     "options"   ? string # options for the loadable module"
     "install"   ? string # command to run when loading module"
-    "remove"    ? string # command to run when removing module "
+    "remove"    ? string # command to run when removing module"
+    "blacklist" ? string # ignored, only the name of the module is used"
```

Comment:
Since the presence of the action keyword is reason enough to process all the other cases, it seemed unnecessary to add an additional check for the value of the "blacklist" action.